package.json

```diff
@@ -5,7 +5,7 @@
   "version": "1.0.1",
   "private": true,
   "scripts": {
-    "dev": "NODE_OPTIONS='--inspect' next dev",
+    "dev": "cross-env NODE_OPTIONS='--inspect' next dev",
```

Comment:
What does this change do @jomarmontuya

Reply:
Hi @ardeay, I recently change to using windows as my development environment and I run into issues where I cannot run the website locally, the reason being is the node options flag don't support windows machine, adding cross-env per nextjs docs will allow to run the node options on any machine,

![image](https://user-images.githubusercontent.com/61284357/186181242-6803e3f4-796e-40dd-9c6e-1c8ff46508c0.png)

Reply:
@ardeay removed please review, thank you!